bin/travis/script.sh

```diff
@@ -19,13 +19,17 @@ if [[ "$ORCA_ENABLE_NIGHTWATCH" == "TRUE" && "$ORCA_SUT_HAS_NIGHTWATCH_TESTS" &&
   (
     cd "$ORCA_YARN_DIR" || exit
     orca fixture:run-server &
-    PID=$!
+    SERVER_PID=$!
+
+    $ORCA_ROOT/vendor/bin/chromedriver --disable-dev-shm-usage --disable-extensions --disable-gpu --headless --no-sandbox --port=4444 &
```

Comment:
There's a bug in the current implementation of `orca qa:automated-tests`: it's supposed to kill Chromedriver on exit, but instead leaves it running. Nightwatch actually depended on this bug in order to run its own tests.

Since we've fixed that in this PR (Chromedriver now exits cleanly along with ORCA), Nightwatch needs to start its own server.